docs/alerts/webhook-connections/opsgenie-legacy.md

```diff
@@ -80,8 +80,9 @@ To create a webhook:
       "SourceURL": "{{SourceURL}}"
     }
     ```
-
-1. Click **Save**.
+1. In the **Recovery Payload** section, you can customize your recovery notification.
```

Comment:
"1. In the **Recovery Payload** section, you can customize your recovery notification." Not applicable to ServiceNow Legacy. Remove it.